gengen/util/gengen.go

```diff
@@ -40,10 +40,16 @@ type Miner struct {
 	// PeerID is the peer ID to set as the miners owner
 	PeerID string
 
-	// Power is the amount of power this miner should start off with
+	// NumCommittedSectors is the number of sectors that this miner has
+	// committed to the network.
+	//
+	// TODO: This struct needs a field which represents the size of sectors
```

Comment:
👍 Could we link an issue saying the same thing within this TODO?

Reply:
Done!